### PR TITLE
🚩(funmooc) split REACT_DASHBOARD feature flag

### PR DESCRIPTION
**Pull request by jbpenrath**
We would like to be able to enable the new dashboard but without updating user menu entries. In order to do that we split `REACT_DASHBOARD` feature flag in two: `REACT_DASHBOARD` which, if it is True, add dashboard urls in Django urls. `REACT_DASHBOARD_MENU_LINKS` which, if it is True, add new dashboard links into user menu.

### CODE REVIEW

File: sites/funmooc/src/backend/funmooc/settings.py [hunk 1]
```diff
@@ -273,13 +273,13 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
...
     if (
-        FEATURES.get("REACT_DASHBOARD", False)  # pylint: disable=no-member
+        FEATURES.get("REACT_DASHBOARD_MENU_LINKS", False)  # pylint: disable=no-member
         and JOANIE_BACKEND.get("BASE_URL") is not None
     ):
         profile_dashboard_urls = {
             "dashboard": {
                 "label": _("Dashboard"),
-                "href": _("/dashboard/"),
+                "href": _("/dashboard"),
```
Comment (kernicPanel) [inline sites/funmooc/src/backend/funmooc/settings.py:282]: Are we sure we need to remove the ending slash ?

Comment (jbpenrath) [reply sites/funmooc/src/backend/funmooc/settings.py:282]: I test it locally. It just to be aligned with the declaration below to the teacher dashboard

Comment (kernicPanel) [reply sites/funmooc/src/backend/funmooc/settings.py:282]: Nice!

**Review by kernicPanel — APPROVED** [review]

File: env.d/development [hunk 2]
```diff
@@ -38,4 +38,4 @@ AUTHENTICATION_BACKEND=dummy
 #JOANIE_BASE_URL=http://localhost:8071
 
 # Features
-FEATURES={'WISHLIST': False, 'REACT_DASHBOARD': False}
+@FEATURES={'WISHLIST': False, 'REACT_DASHBOARD': False, 'REACT_DASHBOARD_MENU_LINKS': False}
```
Comment (lunika) [inline env.d/development:41]:
```suggestion
FEATURES={'WISHLIST': False, 'REACT_DASHBOARD': False, 'REACT_DASHBOARD_MENU_LINKS': False}
```

**Review by lunika — APPROVED** [review]

**Pull request merged**